### ISSUE TITLE
Repl never prompts

### ISSUE THREAD

**Issue by dchacke** [plexus/chestnut#247]
- [x] I am reporting a bug or problem in Chestnut

I create a new project and then try to launch the repl:

```bash
➜ lein new chestnut frontend-two --snapshot +reagent
Generating fresh Chestnut project.
README.md contains instructions to get you started.
➜ cd frontend-two 
➜ lein repl
2019-05-05 01:11:36.976:INFO::main: Logging initialized @2180ms to org.eclipse.jetty.util.log.StdErrLog
[WARNING] No nREPL middleware descriptor in metadata of #'cider.piggieback/wrap-cljs-repl, see clojure.tools.middleware/set-descriptor!
nREPL server started on port 60606 on host 127.0.0.1 - nrepl://127.0.0.1:60606
```

It says that the repl server has started, but I never actually get a prompt. My terminal just keeps waiting (I gave up after a few minutes).

**Comment by featheredtoast** [comment]
Trying to reproduce this - What version(s) of clojure+java is this on?

**Comment by weltschmerz1** [comment]
Java version 1.8.0_111, Clojure version 1.8.0.

**Comment by plexus** [comment]
@weltschmerz1 some things to check:

- is your leiningen the latest version (`lein upgrade`)?
- do you have anything in `~/.lein/profiles.clj` that could be interfering? (`mv ~/.lein/profiles.clj ~/lein/profiles.clj.backup`)

**Comment by plexus** [comment]
Duplicate of #243.

@weltschmerz1 could you also check your current version of leiningen before upgrading (`lein version`), and tell us your operating system?

**Comment by weltschmerz1** [comment]
Upgrading from Leiningen 2.7.1 to 2.9.1 fixed it.

**Comment by dchacke** [comment]
I'm afraid I can now reproduce this again. Leiningen 2.9.1, Java 1.8.0_111, Clojure 1.10.0.

**Comment by dchacke** [comment]
Has anybody had a chance to look into this?

**Comment by featheredtoast** [comment]
Not yet sorry - will look into it soon. Was it the new Clojure version that triggered it this time?

**Comment by dchacke** [comment]
I'm trying it out - I changed the Clojure version in project.clj to 1.8.0 to see if that fixes it, but it doesn't. But that could be because I don't know how to properly downgrade.

Does the project need to be *created* using Clojure 1.8.0? Or should changing it in project.clj after creation do the trick?

**Comment by featheredtoast** [comment]
I don't believe either makes a difference - Clojure version change might be a red herring, but I'll see if I can reproduce it under the same environment.

**Comment by whenmoon** [comment]
Was there ever a fix for this?
